app/workers/rapid_ready_for_decision/asthma_processor.rb

```diff
@@ -33,7 +33,12 @@ def assess_medications(medications)
     end
 
     def med_stats_hash(assessed_data)
-      { medications_count: assessed_data[:medications]&.size }
+      {
+        medications_count: assessed_data[:medications]&.size,
+        asthma_medications_count: assessed_data[:medications].select do |m|
+                                    m[:flagged]
+                                  end.count
```

Comment:
Does this fit in the line-length limit? Also note that you use `size` and `count` -- let's be consistent.
```suggestion
        asthma_medications_count: assessed_data[:medications].select { |m| m[:flagged] }.size
```

Reply:
> Does this fit in the line-length limit?

What do you mean? it passes the lint check as far as I can tell... 

> Also note that you use `size` and `count` -- let's be consistent.

Thanks! Agreed!